buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -909,20 +909,42 @@
       "enabled": true,
       "continueOnError": true,
       "alwaysRun": false,
```

Comment:
I'd expect alwaysrun to be true for these two steps

Reply:
Good call. I changed all the other files too